data/gml_generator.py

```diff
@@ -29,6 +30,251 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         jpype.shutdownJVM()
 
 
+class GMLDataFiller:
+    def __init__(self, gml, iau_version, code):
```

Comment:
On peut mettre des annotations de typage stp ? Ça amène un peu de contexte

```suggestion
    def __init__(self, gml : bytes, iau_version : str, code : str):
```

Reply:
Fixed in d4432d6